### ISSUE TITLE
wrong unexpected-keyword-arg on check_output

### ISSUE THREAD

**Issue by vzaliva** [PyCQA/pylint#4285]
The following code:

```python
import subprocess

lines = subprocess.check_output(["ls",  "-l"], text=True)
print(lines)
```

Triggers the error:

coq-config.py:85:19: E1123: Unexpected keyword argument 'text' in function call (unexpected-keyword-arg)

`text` is documented argument, and actually works (it affects return value of the function):

https://docs.python.org/3/library/subprocess.html#subprocess.check_output

**Comment by Pierre-Sassoulas** [comment]
I just tested that, I cannot reproduce. Are you sure the python version you're using is 3.7 or superior ? If you launch pylint with python 3.6 the ``text`` argument does not exists yet. 

> New in version 3.7: text was added as a more readable alias for universal_newlines.

**Comment by vzaliva** [comment]
I am using 3.8.5. This is the file: https://github.com/vzaliva/coq-config/blob/main/coq-config.py

**Comment by vzaliva** [comment]
```
$ pylint --version
pylint 2.4.4
astroid 2.3.3
Python 3.8.5 (default, Jan 27 2021, 15:41:15) 
[GCC 9.3.0]
```

**Comment by Pierre-Sassoulas** [comment]
Can you try upgrading to pylint 2.7.4 ?

**Comment by vzaliva** [comment]
If there is an easy way to do this? This a production Ubuntu machine where I prefer to stick with stable packages installed via `apt`.

**Comment by Pierre-Sassoulas** [comment]
Well you can use a virtualenv:
```bash
python3 -m venv venv
source venv/bin/activate
pip3 install pylint -U
# ...
rm -rv venv/
```

**Comment by Pierre-Sassoulas** [comment]
I'm going to close as it's fixed on master and probably ``2.7.4``.
